bpf/bpf_host.c

```diff
@@ -1045,6 +1045,20 @@ int to_netdev(struct __ctx_buff *ctx __maybe_unused)
 	 (defined(ENABLE_DSR) && defined(ENABLE_DSR_HYBRID)) || \
 	 defined(ENABLE_MASQUERADE) || \
 	 defined(ENABLE_EGRESS_GATEWAY))
+
+#ifdef NETFILTER_COMPAT_MODE
+	/* When external traffic access in-cluster service through NodePort.
+	 * For the reply packet of NodePort traffic, do reverse DNAT.
```

Comment:
=> "For a packet which is a reply from a local endpoint to a NodePort request, do rev-DNAT."